src/Tokenizer/Tokens.php

```diff
@@ -599,10 +599,10 @@ public function generatePartialCode(int $start, int $end): string
     /**
```

Comment:
how you suggest to make it compatible with lower PHP versions ?

Reply:
we rely on built-in, enabled by default extensions. no custom extensions needed to be manually installed.
we are OK with polyfills, but not sure if there is any for this feature

Reply:
> we rely on built-in, enabled by default extensions

Not really -> https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/blob/v3.71.0/composer.json#L42 (is built-in, but not enabled by default).

But this is one more reason to stop supporting unsupported PHP versions.

Reply:
@kubawerlos , thus i mentioned either enabled by default **or polyfill**

no need to mess with PHP matrix to have this change possible for those on higher PHP versions.